cli/src/main/java/com/devonfw/tools/ide/common/SystemPath.java

```diff
@@ -28,7 +28,9 @@ public class SystemPath {
 
   private final IdeContext context;
 
-  private static final List<String> EXTENSION_PRIORITY = List.of(".exe", ".cmd", ".bat", ".msi", ".ps1", "");
+  private static final List<String> EXTENSION_PRIORITY_WIN = List.of(".exe", ".cmd", ".bat", ".msi", ".ps1", "");
+
+  private static final List<String> EXTENSION_PRIORITY_UNIX = List.of(".exe", "", ".cmd", ".bat", ".msi", ".ps1");
```

Comment:
I just wonder if "" should be placed in the first place, and also, are the extensions that are only for WINDOWS also relevant to the list for UNIX? Does that also affect the WIN list, in the same sense?